### PR TITLE
add contribution guidelines

### PR DESCRIPTION
**Pull request by rkurchin**
Open to any feedback on this stuff! Does the list of tags look good? Am I missing anything?

(obviously still a work in progress, placeholders should be obvious)

### CODE REVIEW

File: examples/4_sgcnn/sgcnn.jl [hunk 1]
```diff
@@ -10,18 +10,17 @@ using Random
 using ChemistryFeaturization
 using AtomicGraphNets
 using Serialization
-<<<<<<< HEAD
+<< << << < HEAD
```
Comment (thazhemadam) [inline examples/4_sgcnn/sgcnn.jl:13]: Should this be here? Looks like this was overlooked while resolving a merge conflict...

Comment (rkurchin) [reply examples/4_sgcnn/sgcnn.jl:13]: haha yuuup thought I'd fixed that, oops

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+| `priority`         | Indicates an issue that is high-priority (generally only to be used by maintainers)                                            |
+| `question`         | Indicates that an issue or pull request needs more information                                                                 |
+| `wontfix`          | Indicates that work won't continue on an issue or pull request                                                                 |
+
+* If you are adding/changing features, make sure to add/update tests (DO NOT comment out tests!) and documentation accordingly! Ideally, if relevant, include example usage.
+* Keep things modular! If you are fixing/adding multiple things, do so via separate issues/PR's to streamline review and merging.
+* It is recommended that you set up [JuliaFormatter](https://domluna.github.io/JuliaFormatter.jl/dev/) (also see [VSCode extension](https://marketplace.visualstudio.com/items?itemName=singularitti.vscode-julia-formatter)). A GitHub action will check that code adheres to the style guide.
+
+## Getting Started
```
[the remark this answers is not in the document]
Comment (rkurchin) [reply CONTRIBUTING.md:41]: I guess the answer to this depends upon how many people you think are out there who can write decent Julia code, might be interested in contributing some of their code to a package, but haven't actually developed packages before. Given that I was such a person relatively recently (and was one in Python a couple years before that), I tend to think there are a fair number, but I may be biased due to personal experience... ;) At any rate, I don't think it hurts to have it there...

[the remark this answers is not in the document]
Comment (rkurchin) [reply CONTRIBUTING.md:41]: Rather than actually writing a new guide, I decided to just compile some other nice resources.

File: README.md [hunk 3]
```diff
@@ -8,9 +8,11 @@ Documentation is in progress [over here](https://aced-differentiate.github.io/At
 
 ## Getting Started
 
-1. Clone this package to wherever you want to play.
+1. To install the latest tagged version, in your Julia REPL, do `]install AtomicGraphNets`. However, you can also play with the latest version on the `main` branch by skipping to step 2 and then doing `]install /path/to/repo` where you replace the dummy path with the location of your clone.
```
Comment (thazhemadam) [inline README.md:12]: `] install`? shouldn't this be `] add` or is there something I'm missing?

Comment (rkurchin) [reply README.md:12]: hahaha yup oops

**Review by thazhemadam — APPROVED** [review]

**Pull request merged**